libs/storybook/.storybook/preview.ts

```diff
@@ -0,0 +1,84 @@
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { UiModule } from '@egov/cvi-ng';
+import { setCompodocJson } from '@storybook/addon-docs/angular';
+import { INITIAL_VIEWPORTS } from '@storybook/addon-viewport';
+import { applicationConfig, moduleMetadata, Preview } from '@storybook/angular';
+import { StorybookModule } from '../src/lib/storybook.module';
+import docJson from './documentation.json';
+import 'cypress-storybook/client';
+
+setCompodocJson(docJson);
+
+const preview: Preview = {
+  parameters: {
+    layout: 'centered',
+    viewport: {
+      viewports: INITIAL_VIEWPORTS,
+    },
+    controls: { expanded: true },
+    docs: {
+      canvas: { sourceState: 'shown' },
+    },
+    backgrounds: {
+      default: 'Default',
+      values: [
+        {
+          name: 'Default',
+          value: '#ffffff',
+        },
+        {
+          name: 'Dark',
+          value: '#13100d',
+        },
+        {
+          name: 'Gray',
+          value: '#f0f0f2',
+        },
+        {
+          name: 'Darkgray',
+          value: '#cccccc',
+        },
+      ],
+    },
+    options: {
+      storySort: {
+        method: 'alphabetical',
+        order: [
```

Comment:
Defined the `order` array to keep the overall structure of the stories order the same.